packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue

```diff
@@ -113,7 +113,7 @@
 								firstRow,
 								firstRow + MAX_NUMBER_OF_ROWS
 							)"
-							:key="firstRow + index"
+							:key="intervention.id"
```

Comment:
Is is the heart of the fix? If so we might want to just generate a better surrogate key. The idea of `firstRow + index` is basically the same thing as the `uuidv4()`. And I think preferably we would do one way or another, and not trying to juggle two different semantics that are both trying to introduce uniqueness to the same set of items.